medusa/tv/cache.py

```diff
@@ -124,9 +106,9 @@ def __init__(self, provider_id):
                     b'    time NUMERIC)'
                 )
         except Exception as error:
-            logger.log('Error while searching {provider_id}, skipping: {e!r}'.
-                       format(provider_id=provider_id, e=error), logger.DEBUG)
-            logger.log(traceback.format_exc(), logger.DEBUG)
+            log.debug('Error while searching {provider_id}, skipping: {error!r}',
+                      {'provider_id': provider_id, 'error': error})
+            log.debug(traceback.format_exc())
```

Comment:
tbh there's lots of logs I would have changed, but I was more concerned about changing format than rewriting them all.  You're right tho, this should probably use exc_info and combine.